config/webpack.js

```diff
@@ -5,6 +5,7 @@ const webpack = require('webpack');
 const distPath = path.join(__dirname, '../dist');
 
 const config = {
+  mode: 'production',
```

Comment:
Out of curiosity, what does this mode mean?

Reply:
In this configuration it doesn't do anything. It's just required by webpack itself (config linting). It could be used to change the default behavior/config but requires an additional code that was not added here.